Configurations.md

```diff
@@ -391,6 +401,7 @@ Indentation of chain
 
 - **Default value**: `"Block"`
 - **Possible values**: `"Block"`, `"Visual"`
+- **Stable**: ?
```

Comment:
This configuration option (`chain_indent`) wasn't in the `create_config!` macro (or I missed it).

Reply:
It has been removed

Reply:
Could you remove this section from the doc please?